src/components/SchemaView/SchemaSidebarEditView.vue

```diff
@@ -234,6 +248,23 @@ export default {
     getColor(label) {
       return this.settingsStore.colorForLabel(label);
     },
+    enterRenameMode() {
+      this.renameTable = true;
+    },
+    saveNewTableName() {
+      if (this.currLabel === this.label) {
```

Comment:
This should not be required either. If the name does not change, you do not need to show the buttons (which means this is not possible).